src/test/java/org/jitsi/meet/test/base/Participant.java

```diff
@@ -543,4 +543,23 @@ public void saveHtmlSource(File outputDir, String fileName)
      * @return a representation of the filmstrip of this participant.
      */
     public abstract Filmstrip<? extends Participant> getFilmstrip();
+
+    /**
+     * @return a list of representation of the remote participants thumbnails.
+     */
+    public abstract List<RemoteParticipant<T>> getRemoteParticipants();
```

Comment:
Should this method be renamed to include something about it being a thumbnail representation?

Reply:
Any ideas ... do we have something similar on the other page objects?

Reply:
There really isn't a good existing precedent for interactions with remote participant thumbnails; from what I know MeetUIUtils is still used, which takes in an endpoint id. Existing page objects for the most part deal with UI outside of the filmstrip and not dynamically generated like the remote thumbnails.

I think it comes down to what you want the RemoteParticipant to represent. Do you want it to be more of an encompassing class that includes remote participant data, and maybe other helper methods related to the data, as well as DOM interactions? I'm feeling like right now you want it to instead represent the thumbnail itself, considering the javadoc.

Reply:
Then change the javadoc not to say it's returning a representation of the thumbnail?